apps/fortuna/src/eth_utils/utils.rs

```diff
@@ -0,0 +1,66 @@
+use {
+    anyhow::{anyhow, Result},
+    ethers::{contract::ContractCall, middleware::Middleware},
+    std::sync::Arc,
+    tracing,
+};
+
+pub async fn send_and_confirm<A: Middleware>(contract_call: ContractCall<A, ()>) -> Result<()> {
```

Comment:
Note: the function signatures of this function and the one below changed very slightly so that they don't depend on the pyth-specific type declarations in the chain/ module.